front/app/modules/commercial/insights/admin/components/TopBar/TopBar.test.tsx

```diff
@@ -73,26 +102,51 @@ describe('Insights Top Bar', () => {
     render(<TopBar />);
     expect(screen.getByTestId('insightsTopBar')).toBeInTheDocument();
   });
+
   it('renders View name correctly', () => {
     render(<TopBar />);
     expect(
       screen.getByText(mockViewData.data.attributes.name)
     ).toBeInTheDocument();
   });
-  it('renders Project button with correct slug', () => {
+
+  it('if only one project: renders Project button with correct slug', () => {
     render(<TopBar />);
     expect(screen.getByTestId('insightsProjectButton')).toBeInTheDocument();
+    expect(
+      screen.queryByTestId('insightsProjectDropdown')
+    ).not.toBeInTheDocument();
     expect(screen.getByRole('link')).toHaveAttribute(
       'href',
       '/en/projects/test'
     );
   });
+
+  it('if multiple projects: renders Project dropdown with correct content', async () => {
```

Comment:
Thank you for continuing to maintain the tests here 🙌

Reply:
My pleasure!